Hi3Helper.Core/Lang/en.json

```diff
@@ -438,79 +438,79 @@
     "ExtremeGraphicsSettingsWarnTitle": "Very High Setting Preset Selected!",
-    "ExtremeGraphicsSettingsWarnSubtitle": "You are about to set the setting preset to Very High!\r\nVery High setting preset is basically 2x Render Scale with MSAA Enabled and it's VERY UNOPTIMIZED! (unless you have a NASA Supercomputer to run it).\r\n\r\nAre you sure you want to use this setting?",
+    "ExtremeGraphicsSettingsWarnSubtitle": "You are about to set the setting preset to Very High!\r\nVery High setting preset is essentially a 2x Render Scale with MSAA Enabled and is VERY UNOPTIMIZED! (unless you have a NASA Supercomputer to run it).\r\n\r\nAre you sure you want to use this setting?",
     "ExistingInstallTitle": "Existing Installation Detected!",
-    "ExistingInstallSubtitle": "Game has already been installed at this location:\r\n\r\n{0}\r\n\r\nIt is recommended to migrate the game to Collapse Launcher.\r\nHowever, you can still use Official Launcher to start the game.\r\n\r\nDo you want to proceed?",
-    "ExistingInstallBHI3LTitle": "Existing Installation on BetterHI3Launcher Detected!",
-    "ExistingInstallBHI3LSubtitle": "Game has already been installed at this location:\r\n\r\n{0}\r\n\r\nIt is recommended to migrate the game to Collapse Launcher.\r\nHowever, you can still use BetterHi3Launcher to start the game.\r\n\r\nDo you want to proceed?",
-    "ExistingInstallSteamTitle": "Existing Installation on Steam is Detected!",
-    "ExistingInstallSteamSubtitle": "Game has already been installed on Steam at this location:\r\n\r\n{0}\r\n\r\nDo you want to convert this version to the Non-Steam Global version?\nNote: Once you initiate the conversion, you can no longer log-in with your Steam account. Only miHoYo/HoYoverse account login method will be available.\r\n\r\nDo you want to proceed?",
+    "ExistingInstallSubtitle": "Game has already been installed at:\r\n\r\n{0}\r\n\r\nIt is recommended to migrate the game to Collapse Launcher for better support and integration.\r\nRest assured, you can still use Official Launcher to start the game.\r\n\r\nDo you wish to proceed with the migration?",
+    "ExistingInstallBHI3LTitle": "Existing Installation via BetterHI3Launcher Detected!",
+    "ExistingInstallBHI3LSubtitle": "Game has already been installed at:\r\n\r\n{0}\r\n\r\nIt is recommended to migrate the game to Collapse Launcher for better support and integration.\r\nRest assured, you can still use BetterHi3Launcher to start the game.\r\n\r\nDo you want to proceed with the migration?",
+    "ExistingInstallSteamTitle": "Existing Installation via Steam Detected!",
+    "ExistingInstallSteamSubtitle": "Game has already been installed on Steam at:\r\n\r\n{0}\r\n\r\nDo you wish to convert this version to the Non-Steam Global version?\nNote: Once you initiate the conversion, you can no longer log-in with your Steam account. Only the miHoYo/HoYoverse account login method will be available.\r\n\r\nDo you wish to proceed with the conversion and migration?",
```

Comment:
The use of `migration` in this sentence might give double meaning as the account is migrated. Removing it seems a better idea.

Reply:
Agree, We can leave it as "migration" only in this part

Reply:
Fixed.